optuna/visualization/_utils.py

```diff
@@ -99,3 +103,19 @@ def _get_param_values(trials: List[FrozenTrial], p_name: str) -> List[Any]:
     if _is_numerical(trials, p_name):
         return values
     return list(map(str, values))
+
+
+def _filter_nonfinite(trials: List[FrozenTrial]) -> List[FrozenTrial]:
+
+    filtered_trials: List[FrozenTrial] = []
+    for trial in trials:
+        # Not a Number, positive infinity and negative infinity are considered to be non-finite.
+        if trial.value is not None and not np.isfinite(trial.value):
```

Comment:
It seems that this utility assumes that the trial is single-objective optimization, since `trial.value` only supports the single-objective optimization. How about extending this to multi-objective optimization?

Reply:
Done in a16bf53b83a3435af2d321c5121c039cd5087cbd.